localstack-core/localstack/utils/lambda_debug_mode/lambda_debug_mode.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from localstack.aws.api.lambda_ import Arn
+from localstack.utils.lambda_debug_mode.lambda_debug_mode_session import lambda_debug_mode_session
+
+# Specifies the fault timeout value in seconds to be used by time restricted workflows when
+# Debug Mode is enabled. The value is set to one hour to ensure eventual termination of
+# long-running processes.
+DEFAULT_LAMBDA_DEBUG_MODE_TIMEOUT_SECONDS: int = 3_600
+
+
+def is_lambda_debug_mode() -> bool:
+    return lambda_debug_mode_session.is_lambda_debug_mode()
+
+
+def lambda_debug_port_for(lambda_arn: Arn) -> Optional[int]:
+    if not is_lambda_debug_mode():
+        return None
+    debug_configuration = lambda_debug_mode_session.debug_config_for(lambda_arn=lambda_arn)
+    if debug_configuration is None:
+        return None
+    return debug_configuration.debug_port
+
+
+def is_lambda_debug_timeout_for(lambda_arn: Arn) -> bool:
```

Comment:
Would be nice to rename this function to be clearer, and maybe add docstrings to this entire module.

`is_lambda_debug_timeout_for` does not really make sense for me. Maybe `is_lambda_debug_timeout_enabled_for`?

Reply:
changed to `is_lambda_debug_timeout_enforced_for` so it's more consistent with the config flag `enforce_timeout`